berserk/types/challenges.py

```diff
@@ -0,0 +1,119 @@
+from __future__ import annotations
+
+from typing_extensions import TypedDict, NotRequired, TypeAlias, Literal
+
+
+Status: TypeAlias = Literal[
+    "created",
+    "offline",
+    "canceled",
+    "declined",
+    "accepted",
+]
+
+
+Title: TypeAlias = Literal[
+    "GM",
```

Comment:
You should use the ones defined in `.common.py` for Title, Colors etc, instead of redefining them here